src/main/java/org/betonquest/betonquest/modules/versioning/java/JREVersionPrinter.java

```diff
@@ -0,0 +1,61 @@
+package org.betonquest.betonquest.modules.versioning.java;
+
+import lombok.CustomLog;
+
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+/**
+ * Utility class for getting information about the Java Runtime Environment version.
+ */
+@CustomLog
+public final class JREVersionPrinter {
+
+    /**
+     * The version of the JRE.
+     */
+    private final String version;
+
+    /**
+     * The build number of the JRE.
+     */
+    private final String build;
+    /**
+     * Optional information about the version of the JRE, may not be provided.
+     */
+    private final String optional;
+    /**
+     * Information about the preRelease version of the JRE, may not be provided.
+     */
+    private final String preReleaseInfo;
+    /**
+     * The vendor of the JRE.
+     */
+    private final String vendor;
+
+    /**
+     * Prints the Java Runtime Environment version to the console.
+     */
+    public JREVersionPrinter() {
```

Comment:
This comment is lying. It just prepares the object that can generate a human readable message.